src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py

```diff
@@ -410,8 +410,20 @@ def listDiskUsageOverThreshold(config, updateDB):
 
     if updateDB and not t0Flag:
         agentDrainMode = bool(len(overThresholdDisks))
+        updateWMConfig = False
         if agentConfig and (agentDrainMode != agentConfig["AgentDrainMode"]):
-            reqMgrAux.updateWMAgentConfig(config.Agent.hostName, {"AgentDrainMode": agentDrainMode},
+            
+            if agentConfig["UserDrainMode"] == 'True':
+                if agentDrainMode == 'False':
+                    print("Agent is already in UserDrainMode: True mode. No need to update.")
```

Comment:
Can you please replace the print statement by the logging module (I'd suggest info level)?

Reply:
modified to use logging at `INFO` level.